tasks/go_test.py

```diff
@@ -733,3 +737,145 @@ def parse_test_log(log_file):
                 if f'{json_line["Package"]}/{json_line["Test"]}' in failed_tests:
                     failed_tests.remove(f'{json_line["Package"]}/{json_line["Test"]}')
     return failed_tests, n_test_executed
+
+
+@task
+def get_impacted_packages(ctx, build_tags=None):
+    dependencies = create_dependencies(ctx, build_tags)
+    files = get_modified_files(ctx)
+    print(files)
+    modified_packages = {
+        f"github.com/DataDog/datadog-agent/{os.path.dirname(file)}"
+        for file in files
+        if file.endswith(".go") or file.endswith(".mod") or file.endswith(".sum")
+    }
+    print(modified_packages)
+    imp = find_impacted_packages(dependencies, modified_packages)
+    print("imp", imp)
```

Comment:
```suggestion
    print("Impacted packages:", imp)
```

Reply:
I will rather remove all the useless prints. I think I mostly needed them for debug